common/daemonsets_utils.go

```diff
@@ -8,7 +8,8 @@ import (
 )
 
 const (
-	PriorityClassName = "openshift-user-critical"
+	PriorityClassName  = "openshift-user-critical"
+	kubeRBACProxyImage = "quay.io/openshift/origin-kube-rbac-proxy:latest"
```

Comment:
I think hardcoding this image will cause problems. Among other things - this will probably break in offline environments. Also `origin-kube-proxy` is an image that is part of OCP payload  and hence I wonder if we can refer to it in LSO.

Reply:
now using image as env similar to diskmaker and provisioner in the operator CR.

Reply:
I don't completely understand your second concern about using `origin-kube-proxy`.  Should we use the upstream image?